refresh.template.py

```diff
@@ -50,6 +51,12 @@ def _get_headers(compile_args: typing.List[str], source_path_for_sanity_check: t
     # As an alternative approach, you might consider trying to get the headers by inspecing the Middlemen actions in the aquery output, but I don't see a way to get just the ones actually #included--or an easy way to get the system headers--without invoking the preprocessor's header search logic.
         # For more on this, see https://github.com/hedronvision/bazel-compile-commands-extractor/issues/5#issuecomment-1031148373
 
+    if "cl.exe" in compile_args[0]:
+        return _get_headers_msvc(compile_args)
+    return _get_headers_gnu(compile_args, source_path_for_sanity_check)
+
+
+def _get_headers_gnu(compile_args: typing.List[str], source_path_for_sanity_check: typing.Optional[str] = None):
```

Comment:
Sorry, I was unable to infer the right order of functions.
gcc vs. gnu: happy with both and I don't have a strong opition.

Will change it :)

Reply:
No, no. Sorry to be wasting time on these minor things. Again normally I'd just propose changes. But in the absence of having permissions I think should still say something, since we're working together for a while here.